jupyterhub/services/auth.py

```diff
@@ -352,34 +362,45 @@ def get_login_url(self):
         """Return the Hub's login URL"""
         return self.hub_auth.login_url
 
-    def check_hub_user(self, user_model):
-        """Check whether Hub-authenticated user should be allowed.
+    def check_hub_user(self, model):
+        """Check whether Hub-authenticated user or service should be allowed.
 
         Returns the input if the user should be allowed, None otherwise.
 
         Override if you want to check anything other than the username's presence in hub_users list.
 
         Args:
-            user_model (dict): the user model returned from :class:`HubAuth`
+            model (dict): the user or service model returned from :class:`HubAuth`
```

Comment:
no, user or service. A request can't be identified as a group (at this point, at least), but it can be a *member* of a group (user).